quick_tour/the_big_picture.rst

```diff
@@ -43,7 +44,7 @@ Actions and Controllers
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 Open the ``src/AppBundle/Controller/DefaultController.php`` file and you'll
-see the following code (for now, don't look at the ``@Route`` configuration
+see the following code (for now, don't look at the ``@Route()`` configuration
```

Comment:
I'm -1 on this change, annotations are like classes. We use the `()` to indicate functions